src/components/grid-aware/StatsBlock/StatsBlock.tsx

```diff
@@ -19,17 +19,69 @@ const StatCard = ({ number, secondaryNumber, statement }: StatCardProps) => (
   </div>
 );
 
+type AxisProp = {
+  x: string | number;
+  y: number;
+};
+
+type BarStatProps = {
+  axes: AxisProp[];
```

Comment:
If you go with my suggestion to rename `AxisProp` to `Bar`, then this prop can also be renamed to `bars`.